h/views/activity.py

```diff
@@ -74,6 +76,8 @@ def user_link(userid):
         'total': result.total,
         'user_link': user_link,
         'username_from_id': username_from_id,
+        # The message that is displayed (only) if there are no search result.
+        'zero_message': _('No annotations matched your search.'),
```

Comment:
I thought it was simpler if this value was always present, regardless of whether there's search results or not. The template only renders it if there's no search results.